eogrow/pipelines/batch_to_eopatch.py

```diff
@@ -110,9 +110,9 @@ def _get_output_features(self) -> List[FeatureSpec]:
 
     def build_workflow(self) -> EOWorkflow:
         """Builds the workflow"""
-        userdata_node = None
+        last_node = None
```

Comment:
just for the sake of consistency, we tend to use `previous_node` (since `last` slightly implies it is the last node of the workflow)

Reply:
fixed